model.py

```diff
@@ -37,32 +37,33 @@ def __init__(self, config):
             torch.zeros(config.l_q, 1, 3, 3), requires_grad=True)
         self.sm = nn.Softmax(dim=1)
 
-    def forward(self, X, S1, S2, config):
-        h = self.h(X)
-        r = self.r(h)
-        q = self.q(r)
+    def forward(self, input_view, state_x, state_y, k):
+        """
+        :param input_view: (batch_sz, imsize, imsize)
+        :param state_x: (batch_sz,), 0 <= state_x < imsize
+        :param state_y: (batch_sz,), 0 <= state_y < imsize
+        :param k: number of iterations
+        :return: logits and softmaxed logits
+        """
+        h = self.h(input_view)  # intermediate output
```

Comment:
nit: Please capitalize the first letter for the new comments

Reply:
That's a good point! Fixed it.